lib/idl/idl_checkers.es6.js

```diff
@@ -320,4 +320,26 @@ Right: ${stringify(ra)})`);
       }
     }
   },
+  function leftAddsNonStandardAPIs(logger, left, right) {
+    if (!left.members) return;
+    const name = left.name || left.implementer;
+    if (!right.members) {
+      logger.warn(`${name}: Right has no members`);
+      return;
+    }
+    const lms = left.members;
+    const rms = right.members;
+    var nonstdNames = '';
```

Comment:
Could we use an array for `nonstdNames`? Then we can use `nonstdNames.push(memberName)` in the loop, `nonstdNames.join(', ')` in reporting, and drop the `nonstdNames && ` from the beginning of existence checks.